.github/workflows/main.yml

```
@@ -0,0 +1,17 @@
+name: CI
+on:
+  - push
+  - pull_request
```

No need to run CI on pushes to non-master branches
```suggestion
  push:
    branches: master
  pull_request:
```

---

You’re right 👍🏻